.github/workflows/builder_bazel_slsa3.yml

```diff
@@ -35,6 +35,18 @@ on:
         required: false
         type: string
         default: ""
+      env-image:
```

Comment:
let's re-use the naming from the container-based builder for consistency, unless there's a good reason not to